### PR TITLE
Minor GDAX fix and update to package 0.5.x

### PR DESCRIPTION
**Pull request by werkkrew**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Bug fix / feature

* **What is the current behavior?** (You can also link to an open issue here)
Using GDAX gives an error related to the new import changes.

* **What is the new behavior (if this is a feature change)?**
Fixed the minor bug and also updated to support/require official GDAX 0.5.x module (coinbase/gdax-node) which required a couple of minor changes to the api calls to clear out deprecation warnings.

* **Other information**:

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,7 +29,7 @@
     "co-fs": "^1.2.0",
     "coinfalcon": "^1.0.3",
     "commander": "^2.9.0",
-    "gdax": "^0.4.2",
+    "gdax": "github:coinbase/gdax-node",
```
Comment (askmike) [inline package.json:32]: I think this requires git to be installed, Gekko's installation (using npm) should also work without Git. I'll check this asap.

**Review by askmike — COMMENTED** [review]
see comment.

File: package.json (hunk 1 above)
Comment (werkkrew) [reply package.json:32]: You might be right, I installed it this way based on information in their official repository.  The command was npm install coinbase/gdax-node. 

I am not sure why they haven't updated the other npm package.

**Comment by werkkrew** [comment]
Closing as the primary issue I was trying to fix is resolved and updating to gdax-node from git is not ideal.

**Pull request closed without merging**